src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -1009,6 +1011,32 @@ public async Task TestQueryFilterNestedFieldAuth_UnauthorizedNestedField()
             Assert.IsTrue(errorMessage.Contains(DataApiBuilderException.GRAPHQL_FILTER_FIELD_AUTHZ_FAILURE));
         }
 
+        [TestMethod]
```

Comment:
need comment elaborating on what is actually being validate: which nested query filter field is disallowed?

Reply:
Added summary on the Test. I didn't put `display name` intentionally, because in this file, none of the test has display name.

Reply:
makes sense. We only use `DisplayName` for `[DataRow]` tests.

Reply:
The updated comment here is very helpful :)